pandas/core/resample.py

```diff
@@ -1025,6 +1025,10 @@ class TimeGrouper(Grouper):
     Use begin, end, nperiods to generate intervals that cannot be derived
     directly from the associated object
     """
+    _attributes = Grouper._attributes + ('closed', 'label', 'how',
+                                         'nperiods', 'fill_method', 'limit',
```

Comment:
remove nperiods, fill_method, and limit

Reply:
and remove nperiods from the signature entirely, its is not used

Reply:
Ok, but what's the benefit of not having fill_method and limit in _attributes?

Reply:
because they are deprecated (see the maybe_process_deprecations)

Reply:
Also, there's this 
```
    Notes
    -----
    Use begin, end, nperiods to generate intervals that cannot be derived
    directly from the associated object
```

None of begin or end seem to exist, and if nperiods isn't used, I've just removed that note

Reply:
yeah remove that, looks bogus